test/src/player.spec.js

```diff
@@ -2857,6 +2857,32 @@ describe('Player', function() {
     });
   });
 
+  describe('on resize event', function() {
+    let config, player, sandbox;
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
+      config = getConfigStructure();
+      config.sources = sourcesConfig.Mp4;
+      player = new Player(config);
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+      player.destroy();
+    });
+
+    it('should call _resetTextCuesAndReposition function', done => {
```

Comment:
In general I think the correct title should be more declarative of what is happening, e.g. - 

> should re-calculate size and position of text cue

Reply:
but it just calls that function and that's what the unit test checks.

there is another unit test that checks the _resetTextCuesAndReposition function itself.